drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -620,6 +674,224 @@ def self.sections(report) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticC
+          required_for: 'HoH in HP',
+          detail_columns: default_detail_columns + [
+            :target_screen_required,
+            :target_screen_completed,
+          ],
+          denominator: ->(item) { hoh?(item) && HudUtility2024.performance_reporting[:prevention].include?(item.project_type) },
+          limiter: ->(item) {
+            return false unless hoh?(item)
+            return false unless HudUtility2024.performance_reporting[:prevention].include?(item.project_type)
+            # Screening Required flag is missing
+            return true if item.target_screen_required.blank?
+            # Screening is not required
+            return false if item.target_screen_required == 0
+
+            !item.target_screen_completed
+          },
+        },
+        vamc_station: {
+          title: 'VAMC Station Number is Missing',
+          description: 'VAMC Station Number is missing or not collected',
+          required_for: 'HoH and Veteran in ES (E/E), TH, PSH, SO, SH, PH-Housing Only, and RRH',
+          detail_columns: default_detail_columns + [
+            :vamc_station,
+          ],
+          denominator: ->(item) {
+            (hoh?(item) || item.veteran) && [
+              HudUtility2024.performance_reporting[:es_entry_exit],
+              HudUtility2024.performance_reporting[:th],
+              HudUtility2024.performance_reporting[:psh],
+              HudUtility2024.performance_reporting[:so],
+              HudUtility2024.performance_reporting[:sh],
+              HudUtility2024.performance_reporting[:oph],
+              HudUtility2024.performance_reporting[:rrh],
+            ].flatten.include?(item.project_type)
+          },
+          limiter: ->(item) {
+            return false unless hoh?(item) || item.veteran
+            return false unless [
+              HudUtility2024.performance_reporting[:es_entry_exit],
+              HudUtility2024.performance_reporting[:th],
+              HudUtility2024.performance_reporting[:psh],
+              HudUtility2024.performance_reporting[:so],
+              HudUtility2024.performance_reporting[:sh],
+              HudUtility2024.performance_reporting[:oph],
+              HudUtility2024.performance_reporting[:rrh],
+            ].flatten.include?(item.project_type)
+
+            item.vamc_station.blank?
+          },
+        },
+        entering_from_streets_issues: {
+          title: 'Client Entering From Streets, ES, or SH',
+          description: 'Prior living situation is "Data not collected" (99) or blank',
+          required_for: 'Adults and HoH',
+          detail_columns: default_detail_columns + [
+            :living_situation,
+          ],
+          denominator: ->(item) { hoh_or_adult?(item) },
+          limiter: ->(item) {
+            return false unless hoh_or_adult?(item)
+
+            item.living_situation.blank? || item.living_situation == 99
+          },
+        },
```

Comment:
Pretty sure this is already captured in `living_situation_issues`.  Let's adjust that logic a bit to exclude 99 from the current check, which currently counts it as acceptable I think.